server/index.js

```diff
@@ -35,12 +37,10 @@ let io = socket(server);
 
 io.on("connection", function (client) {
 	console.log("Client[" + client.id + "] connected");
-	clients[client.id] = {
-		roomCode: null,
-		joined_room: false
-	}
+	let roomCode;
+	let code;
```

Comment:
Do we need both roomCode and code?  I forgot if there was a reason for this or if it's even necessary anymore.